File: app/src/main/java/rrzaniolo/testandroidsantander/network/investment/response/InvestmentResponse.java
package rrzaniolo.testandroidsantander.network.investment.response;

/*
 * Created by Rodrigo Rodrigues Zaniolo on 5/9/2018.
 * All rights reserved.
 */

import com.google.gson.annotations.SerializedName;

import rrzaniolo.testandroidsantander.network.investment.models.Screen;

/**
 * Model for the Investment Api response.
 * */
public class InvestmentResponse {
    //region --- Variable
    @SerializedName("screen")
    private Screen screen;
    public Screen getScreen() {
        return screen;
    }
    public void setScreen(Screen screen) {
        this.screen = screen;
    }
    //endregion

    //region --- Constructors
    public InvestmentResponse() { }

    public InvestmentResponse(Screen screen) {
        this.screen = screen;
    }
    //endregion
}

File: SantanderInvestimento/app/src/main/kotlin/br/com/santander/santanderinvestimento/feature/investiment/data/entity/InvestmentResponse.kt
package br.com.santander.santanderinvestimento.feature.investiment.data.entity

import br.com.santander.santanderinvestimento.feature.investiment.domain.entity.Investment
import br.com.santander.santanderinvestimento.util.JSONConvertable
import com.google.gson.annotations.SerializedName

class InvestmentResponse(@SerializedName("screen") val screen: Investment) : JSONConvertable